cli/src/commands/account.rs

```diff
@@ -389,11 +410,70 @@ mod tests {
         );
 
         let vanity = None;
-        let account = Account::New { network: 0, seed, vanity, discreet: false };
+        let account = Account::New { network: 0, seed, vanity, discreet: false, save_to_file: None };
         let actual = account.parse().unwrap();
         assert_eq!(expected, actual);
     }
 
+    #[test]
+    fn test_new_save_to_file() {
+        let dir = TempDir::new().expect("Failed to create temp folder");
+        let file = dir.path().join("my-private-key-file").to_string_lossy().to_string();
+
+        let seed = Some(1231275789u64.to_string());
+        let vanity = None;
+        let discreet = false;
+        let save_to_file = Some(file.clone());
+        let account = Account::New { network: 0, seed, vanity, discreet, save_to_file };
+        let actual = account.parse().unwrap();
+
+        let expected = "APrivateKey1zkp2n22c19hNdGF8wuEoQcuiyuWbquY6up4CtG5DYKqPX2X";
+        assert!(actual.contains(expected));
+
+        let content = fs::read_to_string(file).expect("Failed to read private-key-file");
+        assert_eq!(expected, content);
+    }
+
+    #[test]
+    fn test_new_prevent_overwrite_existing_file() {
+        let mut file = NamedTempFile::new().expect("Failed to create temp file");
+        write!(file, "{}", "don't overwrite me").expect("Failed to write secret to file");
```

Comment:
```suggestion
        write!(file, "don't overwrite me").expect("Failed to write secret to file");
```
nit

Reply:
Thanks :sun_with_face:  Fixed in d2d142f6e